### PR TITLE
Add a Rollbar error handler

### PR DESCRIPTION
**Pull request by tomaszkrzywda**
To make the Rollbar integration easier for applications leveraging Eventboss.

### CODE REVIEW

File: lib/eventboss/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Eventboss
-  VERSION = "1.4.1"
+  VERSION = "1.5.0"
```
Comment (nglx) [inline lib/eventboss/version.rb:2]: Oh really, why not `1.4.2`, it's just a small error notification addition 🤣

File: lib/eventboss/error_handlers/rollbar.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Eventboss
+  module ErrorHandlers
+    class Rollbar
+      def call(exception, context = {})
+        eventboss_context = { component: 'eventboss' }
+        eventboss_context[:action] = context[:processor].class.to_s if context[:processor]
+        ::Rollbar.error(exception, eventboss_context.merge(context))
```
Comment (przprz) [inline lib/eventboss/error_handlers/rollbar.rb:7]: It won't work with Ruby 3's explicit kw args I guess ;)

File: lib/eventboss/version.rb (hunk 1 above)
Comment (tomaszkrzywda) [reply lib/eventboss/version.rb:2]: Well, according to the SemVer manifesto: `PATCH version when you make backwards compatible bug fixes.`
But I'm fine with anything here - it's your toy in the end:).

**Review by przprz — APPROVED** [review]

File: lib/eventboss/version.rb (hunk 1 above)
Comment (przprz) [reply lib/eventboss/version.rb:2]: I agree with @tomaszkrzywda and remember we had similar discussion with @nglx with one of previous PRs 😄

Comment (nglx) [reply lib/eventboss/version.rb:2]: convinced.

**Review by nglx — APPROVED** [review]

File: lib/eventboss/error_handlers/rollbar.rb (hunk 2 above)
Comment (przprz) [reply lib/eventboss/error_handlers/rollbar.rb:7]: I mean the 2nd param is a Hash. 
Nevermind, maybe it's not an issue now.

File: lib/eventboss/error_handlers/rollbar.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module Eventboss
+  module ErrorHandlers
+    class Rollbar
+      def call(exception, context = {})
```
Comment (wkosak) [inline lib/eventboss/error_handlers/rollbar.rb:4]: @tomaszkrzywda just received Dependabot notification about this change 👏 

What do you think about sending `context[:processor].jid` to Rollbar as well? We use `jid` to match errors with Kibana logs:

https://github.com/AirHelp/ahplus/blob/8216b5357797e5ee9d193ead58be9870f7ff706e/config/initializers/eventboss.rb#L5

Comment (tomaszkrzywda) [reply lib/eventboss/error_handlers/rollbar.rb:4]: @wkosak I can't think of any reason not to do it:). Rollbar.error accepts additional data as well.

**Pull request merged**